src/functionalities.js

```diff
@@ -0,0 +1,120 @@
+import updateStatus from './status.js';
+
+export default class Tasks {
+  constructor() {
+    this.tasksList = [];
+  }
+
+  removeTask(index) {
+    this.tasksList.splice(index, 1);
+    for (let i = index; i < this.tasksList.length; i += 1) {
```

Comment:
In this case its advisable to the for each Loop instead

Reply:
The reason I used the `for` loop here instead, was the need for the `i` counter, as I use it to update the indexes of the tasks. 
I did think of using `forEach()`, but the `for` loop was easier to use.